modules/tinymce/src/core/test/ts/browser/FontSelectTest.ts

```diff
@@ -80,6 +80,32 @@ UnitTest.asynctest('browser.tinymce.core.FontSelectTest', function (success, fai
         sAssertSelectBoxDisplayValue(editor, 'Fonts', 'Times')
       ])),
 
+      // https://websemantics.uk/articles/font-size-conversion/
+      Logger.t('TINY-6291: Font size on paragraph with keyword font size is translated to default size', GeneralSteps.sequence([
```

Comment:
We normally try to avoid using `Logger` as it's an old API and largely deprecated (still useful in select cases IIRC). I'm not worried about fixing it for this case, given everything else in this file uses it but just wanted to mention it for future PRs.

The newer version we'd use is this:
```
Log.stepsAsStep('TINY-6291', 'Font size on paragraph with keyword font size is translated to default size', [
```